Study_In_Germany.md

```diff
@@ -0,0 +1,28 @@
+# How the German higer education system works 😄
+
```

Comment:
```suggestion
## Different types of universities
```

Reply:
or - maybe you can drop the "How the German higher education system works and reduce the section level all by one